File: CS/WindowsApplication27/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraSpellChecker;
using System.IO;
using System.Reflection;

namespace WindowsApplication27 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            SpellCheckerISpellDictionary dictionary = new SpellCheckerISpellDictionary();
            Stream affStream = Assembly.GetExecutingAssembly().GetManifestResourceStream("WindowsApplication27.Dictionaries.english.aff");
            Stream dicStream = Assembly.GetExecutingAssembly().GetManifestResourceStream("WindowsApplication27.Dictionaries.american.xlg");
            Stream alphStream = Assembly.GetExecutingAssembly().GetManifestResourceStream("WindowsApplication27.Dictionaries.EnglishAlphabet.txt");
            dictionary.LoadFromStream(dicStream, affStream, alphStream);
            spellChecker1.Dictionaries.Add(dictionary);
        }
    }
}

File: VB/WindowsApplication27/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraSpellChecker
Imports System.IO
Imports System.Reflection

Namespace WindowsApplication27
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim dictionary As New SpellCheckerISpellDictionary()
			Dim affStream As Stream = System.Reflection.Assembly.GetExecutingAssembly().GetManifestResourceStream("english.aff")
			Dim dicStream As Stream = System.Reflection.Assembly.GetExecutingAssembly().GetManifestResourceStream("american.xlg")
			Dim alphStream As Stream = System.Reflection.Assembly.GetExecutingAssembly().GetManifestResourceStream("EnglishAlphabet.txt")
			dictionary.LoadFromStream(dicStream, affStream, alphStream)
			spellChecker1.Dictionaries.Add(dictionary)
		End Sub
	End Class
End Namespace